sdks/python/src/opik/integrations/langchain/opik_tracer.py

```diff
@@ -211,12 +223,17 @@ def _process_end_trace(self, run: "Run") -> None:
             span_data = self._span_data_map[run.id]
             if openai_run_helpers.is_openai_run(run):
                 usage = openai_run_helpers.try_get_token_usage(run_dict)
+                provider, model = openai_run_helpers.get_provider_and_model(run_dict)
```

Comment:
we should unite `try_get_token_usage` and `get_provider_and_model` into one function, let it return something like TokenUsageInfo (example name) with fields `usage`, `provider` and `model`